internal/dcache/cluster_manager/cluster_manager.go

```diff
@@ -2144,21 +2152,24 @@ func (cmi *ClusterManager) updateMVList(rvMap map[string]dcache.RawVolume,
 	// per rvMap, then the component RV is force marked offline. Then it sets the MV state based on the cumulative
 	// state of all of it's component RVs as follows:
 	// - If all component RVs of an MV are online, the MV is marked as online, else
-	// - If no component RV of an MV is online (they are either offline, outofsync or syncing), the MV
-	//   is marked as offline, else
+	// - If no component RV of an MV is online (they are either offline, outofsync, syncing or inband-offline),
+	//   the MV is marked as offline, else
 	// - If at least one component RV is online, the MV is marked as degraded, else
 	// - All component RVs are either online or syncing, then MV is marked as syncing.
 	//
 	// Few examples:
 	// online, online, online => online
 	// online, online, offline => degraded
 	// online, online, outofsync => degraded
+	// online, online, inband-offline => degraded
```

Comment:
nit, instead of a new entry, change L2162 to
online, online, offline|inband-offline => degraded

Replace all lines with offline similarly, and in the end

offline|inband-offline, offline|inband-offline, offline|inband-offline=> offline